### PR TITLE
Create more features1.20.1汉化文件提交

### PR DESCRIPTION
**Pull request by Ballon-zzx**

### CODE REVIEW

**Comment by Ballon-zzx** [comment]
这个模组尚未上传到curseforge里面，只有modrith有

**Comment by dovisutu** [comment]
/mv projects/1.20/assets/create_more_features/create_more_features projects/1.20/assets/modrinth-create_more_features/create_more_features

**Comment by Ballon-zzx** [comment]
> /mv projects/1.20/assets/create_more_features/create_more_features projects/1.20/assets/modrinth-create_more_features/create_more_features

?

**Comment by Ballon-zzx** [comment]
> /mv projects/1.20/assets/create_more_features/create_more_features projects/1.20/assets/modrinth-create_more_features/create_more_features

!感谢，我前面只上传过curseforge上面模组的汉化，modrith还没整过

**Comment by Ballon-zzx** [comment]
> /mv projects/1.20/assets/create_more_features/create_more_features projects/1.20/assets/modrinth-create_more_features/create_more_features

是改成这个格式吗

**Comment by dovisutu** [comment]
/mv projects/1.20/assets/modrinth-create_more_features/create_more_features projects/1.20/assets/modrinth-create-more-features/create_more_features/lang

**Comment by ChuijkYahus** [comment]
/sort-keys projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/en_us.json
/sort-keys projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.cultural_slab": "栽培木活板门",
+  "block.create_more_features.cultural_stairs": "栽培木楼梯",
+  "block.create_more_features.cultural_wood": "栽培木",
+  "block.create_more_features.dimmable_rose_quartz_lamp": "可调节玫瑰石英灯",
+  "block.create_more_features.engineers_desk": "工程师桌",
+  "block.create_more_features.flying_block": "飞行方块",
+  "block.create_more_features.honeycomb_collector": "蜂窝收集器",
+  "block.create_more_features.incomplete_dimmable_rose_quartz_lamp": "不完整的可调节粉红色石英灯",
+  "block.create_more_features.incomplete_railway_decorative_casing": "不完整的铁路装饰机壳",
+  "block.create_more_features.incomplete_signalling": "不完整的信号装置",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:42]:
```suggestion
  "block.create_more_features.incomplete_dimmable_rose_quartz_lamp": "可调节玫瑰石英灯（半成品）",
  "block.create_more_features.incomplete_railway_decorative_casing": "列车装饰机壳（半成品）",
  "block.create_more_features.incomplete_signalling": "信号装置（半成品）",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.pipe_table": "管材桌",
+  "block.create_more_features.railway_casing_slab": "列车机壳台阶",
+  "block.create_more_features.railway_casing_stairs": "列车机壳楼梯",
+  "block.create_more_features.railway_decorative_casing": "铁路装饰机壳",
+  "block.create_more_features.railway_decorative_casing_fence": "铁路装饰栅栏",
+  "block.create_more_features.railway_decorative_casing_fence_gate": "铁路装饰护板栅栏门",
+  "block.create_more_features.railway_decorative_casing_slab": "铁路装饰台阶",
+  "block.create_more_features.railway_decorative_casing_stairs": "铁路装饰外壳楼梯",
+  "block.create_more_features.railway_decorative_casing_trapdoor": "铁路装饰活板门",
+  "block.create_more_features.railway_decorative_casing_wall": "铁路装饰护墙板",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:55]:
```suggestion
  "block.create_more_features.railway_decorative_casing": "列车装饰机壳",
  "block.create_more_features.railway_decorative_casing_fence": "列车装饰机壳栅栏",
  "block.create_more_features.railway_decorative_casing_fence_gate": "列车装饰机壳栅栏门",
  "block.create_more_features.railway_decorative_casing_slab": "列车装饰机壳台阶",
  "block.create_more_features.railway_decorative_casing_stairs": "列车装饰机壳楼梯",
  "block.create_more_features.railway_decorative_casing_trapdoor": "列车装饰机壳活板门",
  "block.create_more_features.railway_decorative_casing_wall": "列车装饰机壳墙",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.refined_radiance_block": "光辉石块",
+  "block.create_more_features.refined_radiance_casing": "光辉石机壳",
+  "block.create_more_features.refined_radiance_casing_slab": "光辉石机壳台阶",
+  "block.create_more_features.refined_radiance_casing_stairs": "光辉石机壳楼梯",
+  "block.create_more_features.seller_of_mechanisms_table": "机械师桌",
+  "block.create_more_features.showcase": "展示柜",
+  "block.create_more_features.signalling": "信号装置",
+  "block.create_more_features.sulfur_basalt_ore": "硫磺玄武岩矿",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:63]:
```suggestion
  "block.create_more_features.sulfur_basalt_ore": "玄武岩硫磺矿石",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.sulfur_lamp": "硫磺灯",
+  "block.create_more_features.sulfur_lamp_up": "明亮的硫磺灯",
+  "block.create_more_features.sulfur_ore": "硫磺矿",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:66]:
```suggestion
  "block.create_more_features.sulfur_ore": "硫磺矿石",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.sundries_table": "杂物桌",
+  "block.create_more_features.velcro": "velcro",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:68]: 未翻译

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.dark_storage": "暗隐水盒",
+  "item.create_more_features.graphite": "石墨",
+  "item.create_more_features.gravitron": "引力仪",
+  "item.create_more_features.gravitron_base": "引力发生器基础",
+  "item.create_more_features.incomplete_diamond": "残缺钻石",
+  "item.create_more_features.incomplete_gravitron": "不完整的引力仪",
+  "item.create_more_features.incomplete_refined_radiance_axe": "未完成的光辉石斧",
+  "item.create_more_features.incomplete_refined_radiance_pickaxe": "未完成的光辉石镐",
+  "item.create_more_features.incomplete_refined_radiance_shovel": "半成品光辉石铲",
+  "item.create_more_features.incomplete_refined_radiance_sword": "半成品光辉石剑",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:93]:
```suggestion
  "item.create_more_features.incomplete_diamond": "钻石（半成品）",
  "item.create_more_features.incomplete_gravitron": "引力仪（半成品）",
  "item.create_more_features.incomplete_refined_radiance_axe": "光辉石斧（半成品）",
  "item.create_more_features.incomplete_refined_radiance_pickaxe": "光辉石镐（半成品）",
  "item.create_more_features.incomplete_refined_radiance_shovel": "光辉石锹（半成品）",
  "item.create_more_features.incomplete_refined_radiance_sword": "光辉石剑（半成品）",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+  "entity.minecraft.villager.create_more_features.sellerofmechanisms": "机械师",
+  "entity.minecraft.villager.create_more_features.sundriesseller": "杂货师",
+  "fluid.create_more_features.netheralavs": "余烬熔岩",
+  "gui.create_more_features.worked.label_work_in_progress": "正在制作中",
+  "item_group.create_more_features.create_more_features_tab": "机械动力：更多村民",
+  "item.create_more_features.coal_pin": "煤粉",
+  "item.create_more_features.dark_storage": "暗隐水盒",
+  "item.create_more_features.graphite": "石墨",
+  "item.create_more_features.gravitron": "引力仪",
+  "item.create_more_features.gravitron_base": "引力发生器基础",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:87]:
```suggestion
  "item.create_more_features.gravitron_base": "引力仪基件",
```
gravitron与上方统一，这个方块在游戏内有什么用途

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+  "entity.minecraft.villager.create_more_features.engineer": "工程师",
+  "entity.minecraft.villager.create_more_features.exchangerofmechanisms": "机械交换器",
+  "entity.minecraft.villager.create_more_features.gizmomaster": "机巧大师",
+  "entity.minecraft.villager.create_more_features.lineman": "电工",
+  "entity.minecraft.villager.create_more_features.pipefitter": "管道工",
+  "entity.minecraft.villager.create_more_features.sellerofmechanisms": "机械师",
+  "entity.minecraft.villager.create_more_features.sundriesseller": "杂货师",
+  "fluid.create_more_features.netheralavs": "余烬熔岩",
+  "gui.create_more_features.worked.label_work_in_progress": "正在制作中",
+  "item_group.create_more_features.create_more_features_tab": "机械动力：更多村民",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:82]: features为何翻译成村民

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.incomplete_refined_radiance_axe": "未完成的光辉石斧",
+  "item.create_more_features.incomplete_refined_radiance_pickaxe": "未完成的光辉石镐",
+  "item.create_more_features.incomplete_refined_radiance_shovel": "半成品光辉石铲",
+  "item.create_more_features.incomplete_refined_radiance_sword": "半成品光辉石剑",
+  "item.create_more_features.netheralavs_bucket": "余烬熔岩桶",
+  "item.create_more_features.pieces_of_graphite": "石墨碎片",
+  "item.create_more_features.refined_radiance": "光辉石",
+  "item.create_more_features.refined_radiance_axe": "光辉石斧",
+  "item.create_more_features.refined_radiance_pickaxe": "光辉石镐",
+  "item.create_more_features.refined_radiance_shovel": "光辉石铲",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:99]:
```suggestion
  "item.create_more_features.refined_radiance_shovel": "光辉石锹",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.refined_radiance_storage": "光辉水盒",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:100]: 这个方块有什么用途

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.refined_radiance_sword": "光辉石剑",
+  "item.create_more_features.saltpeter": "盐酸",
+  "item.create_more_features.shadow_ingot": "暗影石",
+  "item.create_more_features.sulfur": "硫磺",
+  "item.create_more_features.void_storage": "黑暗存储",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:105]: 这个方块有什么用途

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.incomplete_refined_radiance_sword": "半成品光辉石剑",
+  "item.create_more_features.netheralavs_bucket": "余烬熔岩桶",
+  "item.create_more_features.pieces_of_graphite": "石墨碎片",
+  "item.create_more_features.refined_radiance": "光辉石",
+  "item.create_more_features.refined_radiance_axe": "光辉石斧",
+  "item.create_more_features.refined_radiance_pickaxe": "光辉石镐",
+  "item.create_more_features.refined_radiance_shovel": "光辉石铲",
+  "item.create_more_features.refined_radiance_storage": "光辉水盒",
+  "item.create_more_features.refined_radiance_sword": "光辉石剑",
+  "item.create_more_features.saltpeter": "盐酸",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:102]:
```suggestion
  "item.create_more_features.saltpeter": "硝石",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+  "painting.create_more_features.disappointment.author": "我的世界社区",
+  "painting.create_more_features.disappointment.title": "失望",
+  "painting.create_more_features.emotions.author": "建立社区",
+  "painting.create_more_features.emotions.title": "情绪",
+  "painting.create_more_features.lack_of_speed.author": "丹尼尔·怀特",
+  "painting.create_more_features.lack_of_speed.title": "转速不足",
+  "painting.create_more_features.mechanizers_yard.author": "丹尼尔·怀特",
+  "painting.create_more_features.mechanizers_yard.title": "机械师的车间",
+  "painting.create_more_features.mr_fire.author": "丹尼尔·怀特",
+  "painting.create_more_features.mr_fire.title": "火焰先生",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:130]: 画作的名称和作者不翻译，请改回英语

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.railway_decorative_casing_fence": "铁路装饰栅栏",
+  "block.create_more_features.railway_decorative_casing_fence_gate": "铁路装饰护板栅栏门",
+  "block.create_more_features.railway_decorative_casing_slab": "铁路装饰台阶",
+  "block.create_more_features.railway_decorative_casing_stairs": "铁路装饰外壳楼梯",
+  "block.create_more_features.railway_decorative_casing_trapdoor": "铁路装饰活板门",
+  "block.create_more_features.railway_decorative_casing_wall": "铁路装饰护墙板",
+  "block.create_more_features.refined_radiance_block": "光辉石块",
+  "block.create_more_features.refined_radiance_casing": "光辉石机壳",
+  "block.create_more_features.refined_radiance_casing_slab": "光辉石机壳台阶",
+  "block.create_more_features.refined_radiance_casing_stairs": "光辉石机壳楼梯",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:59]:
```suggestion
  "block.create_more_features.refined_radiance_casing": "光辉机壳",
  "block.create_more_features.refined_radiance_casing_slab": "光辉机壳台阶",
  "block.create_more_features.refined_radiance_casing_stairs": "光辉机壳楼梯",
```
建议参照现有译名

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.netheralavs_bucket": "余烬熔岩桶",
+  "item.create_more_features.pieces_of_graphite": "石墨碎片",
+  "item.create_more_features.refined_radiance": "光辉石",
+  "item.create_more_features.refined_radiance_axe": "光辉石斧",
+  "item.create_more_features.refined_radiance_pickaxe": "光辉石镐",
+  "item.create_more_features.refined_radiance_shovel": "光辉石铲",
+  "item.create_more_features.refined_radiance_storage": "光辉水盒",
+  "item.create_more_features.refined_radiance_sword": "光辉石剑",
+  "item.create_more_features.saltpeter": "盐酸",
+  "item.create_more_features.shadow_ingot": "暗影石",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:103]:
```suggestion
  "item.create_more_features.shadow_ingot": "暗影锭",
```
建议

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 16]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.sulfur": "硫磺",
+  "item.create_more_features.void_storage": "黑暗存储",
+  "msg.create_more_features.item_tooltip_cultural_sapling": "栽培树的幼苗是专门为工业树采伐而培育的，可以通过手动折断获得其树叶，此外，这种树的高度仅可达到3个方块，这使其非常适合自动化",
+  "msg.create_more_features.item_tooltip_fluids_storages": "这是一个可重复使用的容器，用于储存水或岩浆，按下【RС】键进行设置（【RС】+【Shift】），这些水箱不能从泵中填充，以保持平衡",
+  "msg.create_more_features.item_tooltip_fluids_storages_lava_level": "熔岩层",
+  "msg.create_more_features.item_tooltip_fluids_storages_water_level": "水位等级",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:109]:
```suggestion
  "msg.create_more_features.item_tooltip_fluids_storages_lava_level": "熔岩含量：",
  "msg.create_more_features.item_tooltip_fluids_storages_water_level": "水含量：",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 17]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.refined_radiance_axe": "光辉石斧",
+  "item.create_more_features.refined_radiance_pickaxe": "光辉石镐",
+  "item.create_more_features.refined_radiance_shovel": "光辉石铲",
+  "item.create_more_features.refined_radiance_storage": "光辉水盒",
+  "item.create_more_features.refined_radiance_sword": "光辉石剑",
+  "item.create_more_features.saltpeter": "盐酸",
+  "item.create_more_features.shadow_ingot": "暗影石",
+  "item.create_more_features.sulfur": "硫磺",
+  "item.create_more_features.void_storage": "黑暗存储",
+  "msg.create_more_features.item_tooltip_cultural_sapling": "栽培树的幼苗是专门为工业树采伐而培育的，可以通过手动折断获得其树叶，此外，这种树的高度仅可达到3个方块，这使其非常适合自动化",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:106]:
```suggestion
  "msg.create_more_features.item_tooltip_cultural_sapling": "栽培树的幼苗是专门为工业伐木而培育的，其树叶可以直接由空手破坏获得。此外，这种树最多只会长到3格高，非常适合自动化。",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 18]
```diff
@@ -0,0 +1,132 @@
...
+  "msg.create_more_features.item_tooltip_fluids_storages": "这是一个可重复使用的容器，用于储存水或岩浆，按下【RС】键进行设置（【RС】+【Shift】），这些水箱不能从泵中填充，以保持平衡",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:107]:
```suggestion
  "msg.create_more_features.item_tooltip_fluids_storages": "这是一个可重复使用的容器，用于储存水或熔岩。右击取出可流体，Shift右击放入流体。这些容器不能通过泵填充，以保持游戏平衡。",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 19]
```diff
@@ -0,0 +1,132 @@
...
+  "msg.create_more_features.item_tooltip_fluids_storages_lava_level": "熔岩层",
+  "msg.create_more_features.item_tooltip_fluids_storages_water_level": "水位等级",
+  "msg.create_more_features.item_tooltip_flying_block": "该方块可以安装在空中，即无需支撑，在各种情况下都非常有用",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:110]:
```suggestion
  "msg.create_more_features.item_tooltip_flying_block": "该方块可以直接在空中放置，无需存在其他方块，在各种情况下都非常有用",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 20]
```diff
@@ -0,0 +1,132 @@
...
+  "msg.create_more_features.item_tooltip_graphite": "一种非常强大的燃料，但只有在堆栈中有超过10根石墨的情况下才会燃烧（作为燃料）！",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:111]:
```suggestion
  "msg.create_more_features.item_tooltip_graphite": "一种非常强大的燃料，但只在同槽位中有超过10根石墨时可燃！",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 21]
```diff
@@ -0,0 +1,132 @@
...
+  "msg.create_more_features.item_tooltip_gravitron": "此物品允许你在生存模式中像在创造模式中一样飞行。在此情况下，该物品的强度将被浪费，我建议对物品进行附魔。该物品可以用阴影锭进行修复！",
+  "msg.create_more_features.item_tooltip_gravitron_tooltip": "按住【Shift】键进行总结",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:113]:
```suggestion
  "msg.create_more_features.item_tooltip_gravitron_tooltip": "按住 [Shift] 可查看概要",
```
create本体译法

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 22]
```diff
@@ -0,0 +1,132 @@
...
+  "item.create_more_features.shadow_ingot": "暗影石",
+  "item.create_more_features.sulfur": "硫磺",
+  "item.create_more_features.void_storage": "黑暗存储",
+  "msg.create_more_features.item_tooltip_cultural_sapling": "栽培树的幼苗是专门为工业树采伐而培育的，可以通过手动折断获得其树叶，此外，这种树的高度仅可达到3个方块，这使其非常适合自动化",
+  "msg.create_more_features.item_tooltip_fluids_storages": "这是一个可重复使用的容器，用于储存水或岩浆，按下【RС】键进行设置（【RС】+【Shift】），这些水箱不能从泵中填充，以保持平衡",
+  "msg.create_more_features.item_tooltip_fluids_storages_lava_level": "熔岩层",
+  "msg.create_more_features.item_tooltip_fluids_storages_water_level": "水位等级",
+  "msg.create_more_features.item_tooltip_flying_block": "该方块可以安装在空中，即无需支撑，在各种情况下都非常有用",
+  "msg.create_more_features.item_tooltip_graphite": "一种非常强大的燃料，但只有在堆栈中有超过10根石墨的情况下才会燃烧（作为燃料）！",
+  "msg.create_more_features.item_tooltip_gravitron": "此物品允许你在生存模式中像在创造模式中一样飞行。在此情况下，该物品的强度将被浪费，我建议对物品进行附魔。该物品可以用阴影锭进行修复！",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:112]:
```suggestion
  "msg.create_more_features.item_tooltip_gravitron": "此物品可让你在生存模式中像创造模式一样飞行，如此会消耗该物品的耐久度，我建议进行附魔。该物品可以用暗影锭进行修复！",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 23]
```diff
@@ -0,0 +1,132 @@
...
+  "msg.create_more_features.item_tooltip_gravitron_tooltip": "按住【Shift】键进行总结",
+  "msg.create_more_features.item_tooltip_lamp": "如果你多次发出红石信号，可调灯会逐渐打开和关闭，脉冲次数越多，闪烁次数越多。",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:114]: 游戏内的实际表现是什么

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 24]
```diff
@@ -0,0 +1,132 @@
...
+  "msg.create_more_features.item_tooltip_lava": "用于创造下界石英，与水相互作用时有1/3的几率生成花岗岩",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:115]:
```suggestion
  "msg.create_more_features.item_tooltip_lava": "用于制造下界岩，与水相互作用时有1/3的概率生成花岗岩",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 25]
```diff
@@ -0,0 +1,132 @@
+{
+  "advancements.dark_matter.descr": "通过用伊拉克拉降落到地牢的深渊中，然后拾取精致的光芒来制造阴影铸块",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:2]:
```suggestion
  "advancements.dark_matter.descr": "携带光辉石时借助鞘翅飞入末地的深渊，以获得暗影锭",
```
翻译后记得检查译文

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 26]
```diff
@@ -0,0 +1,132 @@
...
+  "advancements.dark_matter.title": "暗物质",
+  "advancements.dazzling_beauty.descr": "创造精致的光芒",
+  "advancements.dazzling_beauty.title": "光辉石",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:5]:
```suggestion
  "advancements.dazzling_beauty.descr": "制造一块光辉石",
  "advancements.dazzling_beauty.title": "光彩夺目",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 27]
```diff
@@ -0,0 +1,132 @@
...
+  "advancements.domestication.descr": "种植培育树苗",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:6]:
```suggestion
  "advancements.domestication.descr": "制造一株栽培树苗",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 28]
```diff
@@ -0,0 +1,132 @@
...
+  "advancements.domestication.title": "驯养",
+  "advancements.explosive_things.descr": "收集所有原料制作火药",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:8]:
```suggestion
  "advancements.explosive_things.descr": "收集所有火药原料以制作火药",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 29]
```diff
@@ -0,0 +1,132 @@
...
+  "advancements.explosive_things.title": "爆炸物",
+  "advancements.legal_creativity.descr": "引力制造仪",
+  "advancements.legal_creativity.title": "合法创造",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:11]:
```suggestion
  "advancements.legal_creativity.descr": "制造一台引力仪",
  "advancements.legal_creativity.title": "合法创造",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 30]
```diff
@@ -0,0 +1,132 @@
...
+  "advancements.time_to_shop.descr": "创造任何新的",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:12]:
```suggestion
  "advancements.time_to_shop.descr": "制造任一“机械动力”村民工作方块",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 31]
```diff
@@ -0,0 +1,132 @@
...
+  "block.create_more_features.showcase": "展示柜",
+  "block.create_more_features.signalling": "信号装置",
+  "block.create_more_features.sulfur_basalt_ore": "硫磺玄武岩矿",
+  "block.create_more_features.sulfur_lamp": "硫磺灯",
+  "block.create_more_features.sulfur_lamp_up": "明亮的硫磺灯",
+  "block.create_more_features.sulfur_ore": "硫磺矿",
+  "block.create_more_features.sundries_table": "杂物桌",
+  "block.create_more_features.velcro": "velcro",
+  "block.create_more_features.workshops": "工坊",
+  "entity.minecraft.villager.create_more_features.buyer_of_mechanisms": "零件工",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:70]:
```suggestion
  "entity.minecraft.villager.create_more_features.buyer_of_mechanisms": "构件购买员",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 32]
```diff
@@ -0,0 +1,132 @@
...
+  "entity.minecraft.villager.create_more_features.confectioner": "糖果制造商",
+  "entity.minecraft.villager.create_more_features.contraptionstrader": "大机械师",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:72]:
```suggestion
  "entity.minecraft.villager.create_more_features.contraptionstrader": "机构商",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 33]
```diff
@@ -0,0 +1,132 @@
...
+  "entity.minecraft.villager.create_more_features.engineer": "工程师",
+  "entity.minecraft.villager.create_more_features.exchangerofmechanisms": "机械交换器",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:74]:
```suggestion
  "entity.minecraft.villager.create_more_features.exchangerofmechanisms": "构件交易员",
```

File: projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json [hunk 34]
```diff
@@ -0,0 +1,132 @@
...
+  "entity.minecraft.villager.create_more_features.gizmomaster": "机巧大师",
+  "entity.minecraft.villager.create_more_features.lineman": "电工",
+  "entity.minecraft.villager.create_more_features.pipefitter": "管道工",
+  "entity.minecraft.villager.create_more_features.sellerofmechanisms": "机械师",
```
Comment (ChuijkYahus) [inline projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json:78]:
```suggestion
  "entity.minecraft.villager.create_more_features.sellerofmechanisms": "机构售卖员",
```

**Review by ChuijkYahus — CHANGES_REQUESTED** [review]

**Comment by dovisutu** [comment]
ping @Ballon-zzx 请及时查阅并回应审查意见。

**Comment by luoxueyasha** [comment]
- [7+7](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/issues/2658)时限进入第一阶段，从该评论发出的时间开始计时，届时请注意查看审查意见。若在7天内仍未做出任何实质性回应，则7+7时限自动进入下一阶段。

@Ballon-zzx 


@dovisutu 以后7+7用这套话术，加`即将被搁置`，否则bot不进处理流程

**Pull request closed without merging**